### PR TITLE
test: use correct type for the root partition

### PR DESCRIPTION
**Pull request by mrc0mmand**

### CODE REVIEW

**Review by bluca — APPROVED** [review]

File: test/test-functions [hunk 1]
```diff
@@ -1653,7 +1653,7 @@ create_empty_image() {
     sfdisk "$LOOPDEV" <<EOF
 label: gpt
 type=C12A7328-F81F-11D2-BA4B-00A0C93EC93B name=esp  size=${esp_size}M
-type=0FC63DAF-8483-4772-8E79-3D69D8477DE4 name=root size=${root_size}M bootable
+type=4F68BCE3-E8CD-4DB1-96E7-FBCAF984B709 name=root size=${root_size}M bootable
```
Comment (poettering) [inline test/test-functions]:
```
$ systemd-id128 show 4F68BCE3-E8CD-4DB1-96E7-FBCAF984B709  0FC63DAF-8483-4772-8E79-3D69D8477DE4
NAME          ID                              
root-x86-64   4f68bce3e8cd4db196e7fbcaf984b709
linux-generic 0fc63daf848347728e793d69d8477de4
```

are you sure you want the x86-64 specific root dir here?

maybe use:

```
$(systemd-id128 show root -u | tail -1 | awk '{print $2}')
```

here?

(and i guess we should add `--no-legend` and json output to `systemd-id128` to make this easier to use)

Comment (mrc0mmand) [reply test/test-functions]: Yeah, that's definitely better, thanks.

Comment (poettering) [reply test/test-functions]: ah, sorry, so current versions of systemd-id128 actually support --value/-P. so you can just do:

```
$(systemd-id128 show root -Pu)
```

Comment (mrc0mmand) [reply test/test-functions]: That's even better, nice!

**Pull request merged**